manifests/profiles/openshift.yaml

```diff
@@ -4,7 +4,7 @@ spec:
   components:
     cni:
       enabled: true
-      namespace: kube-system
+      namespace: openshift-operators
```

Comment:
In theory it could work without any changes to this profile, right?

Just the docs need update? With instructions to add the labels to the `kube-system` namespace and add the privileged scc to the istio-cni service account?

Reply:
I also tried

```shell
$ oc adm policy add-scc-to-user privileged  -n kube-system -z istio-cni
```

but that still didn't work for me given that the labels weren't there.

Reply:
have you tried adding the labels to the namespace manually (e.g., kubectl edit ns kube-system)

Reply:
Also, when I added the labels via `kubectl edit` it still doesn't show up with `kubectl get ns kube-system --show-labels`

```yaml
apiVersion: v1
kind: Namespace
metadata:
  annotations:
    openshift.io/sa.scc.mcs: s0:c1,c0
    openshift.io/sa.scc.supplemental-groups: 1000000000/10000
    openshift.io/sa.scc.uid-range: 1000000000/10000
    pod-security.kubernetes.io/audit: privileged
    pod-security.kubernetes.io/enforce: privileged
    pod-security.kubernetes.io/warn: privileged
  creationTimestamp: "2022-11-15T03:45:33Z"
  labels:
    kubernetes.io/metadata.name: kube-system
  name: kube-system
  resourceVersion: "3755261"
  uid: 5786af5e-ed1a-41c4-a8c8-b1a5bde1a4bb
spec:
  finalizers:
  - kubernetes
status:
  phase: Active
```

```shell
$ k get ns kube-system --show-labels
NAME          STATUS   AGE     LABELS
kube-system   Active   6d17h   kubernetes.io/metadata.name=kube-system
```

Reply:
looks like pod security admission admission synchronization is disable on `kube-*` namespace.

Reply:
See https://github.com/istio/istio/issues/42055#issuecomment-1322425546

@morvencao and @jwendell can you please follow up on this with a PM at RedHat and see what they suggest. I can make the changes to the documents. I'm guessing we do this in `istio-system`.